### PR TITLE
Backups - Added backup section to the handbook.

### PR DESCRIPTION
**Pull request by panshulK**
Changes Made - 

- Added backup section to the handbook.

### CODE REVIEW

File: content/docs/common-areas/backups.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: Backups
+weight: 4
+description: Taking regular backups of your website can help you in effectively managing the downtime situation.
+---
+
+# Backups
+
+The saying - "Always have a backup plan" stands true and is an integral part of web application management too. Although a lot of measures can be taken to avoid the downtime of a website but it is hard to predict whether we will ever come across this situation or not. We should be well prepared for anything. Hence, having a back up plan or in other words, taking regular backups of your website can help you in effectively managing the downtime situation. The website backups allows you to back up your site and related database via FTP, SFTP, FTPS, or SSH key to a secure cloud location.
```
Comment (swarad07) [inline content/docs/common-areas/backups.md:9]: Should we also recommend taking backups as part of the process before certain tasks, like deployments, running database updates, migrations, module/core updates, etc?

I think calling out such situations might help.

Comment (panshulK) [reply content/docs/common-areas/backups.md:9]: True @swarad07 !
Thanks for the suggestion here. I have added a separate section to highlight these situations.

Please review and let me know if anything else needs to be added here.

Thanks.

File: content/docs/common-areas/_index.md [hunk 2]
```diff
@@ -27,7 +27,9 @@ Most systems have multiple layers of caching, each of which would be suitable to
 
 APM or Application Performance Monitoring can be defined as a process or a practice of monitoring and tracking the performance of a software application using specific tools. The performance of a Drupal application can be monitored using various tools such as NewRelic, Elastic APM, Sentry etc..
 
-## Backups
+## [Backups](/docs/common-areas/backups)
```
Comment (hussainweb) [inline content/docs/common-areas/_index.md:30]:
```suggestion
## [Backups]({{< relref "/docs/common-areas/backups" }}>)
```

We need to use relref for internal links or they may break.

File: content/docs/common-areas/_index.md [hunk 3]
```diff
@@ -27,7 +27,9 @@ Most systems have multiple layers of caching, each of which would be suitable to
...
+
+The saying - "Always have a backup plan" stands true and is an integral part of web application management too. Although a lot of measures can be taken to avoid the downtime of a website but it is hard to predict the situation. We should be well prepared for anything. Hence, having a back up plan or in other words, taking regular backups of your website can help you manage the downtime situation effectively. The website backups allows you to back up your sites and related databases via FTP, SFTP, FTPS, or SSH key to a secure cloud location.
```
Comment (hussainweb) [inline content/docs/common-areas/_index.md:32]: The last sentence doesn't make sense to me.

File: content/docs/common-areas/backups.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+## Importance of taking the backup of your site
+
+These are some of the alarming cases that can be mitigated if the site resources are regularly backed up -
```
Comment (hussainweb) [inline content/docs/common-areas/backups.md:13]:
```suggestion
These are some of the alarming cases that can be mitigated if the site resources are regularly backed up:
```

File: content/docs/common-areas/backups.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+- Incorrect database updates after a release.
+- Accidental content deletion.
+- Virus Attacks.
```
Comment (hussainweb) [inline content/docs/common-areas/backups.md:17]:
```suggestion
- Virus attacks.
```

File: content/docs/common-areas/backups.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+- Server issues.
+- Memory Usage issues.
```
Comment (hussainweb) [inline content/docs/common-areas/backups.md:19]: I think you're stretching here. :) It's okay to leave them out.

File: content/docs/common-areas/backups.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+## Types of website backups -
+
+### Full backup
+
+In this type of of backup all the information & resources of a website are copied over to the secondary target storage. The full backup ensures that files, database, scripts are backed up all the once. Generally, it is recommended to take the backup of the site at the regular intervals. For ex, some servers use automated scripts to perform the backup at set day & time. Based upon the scale of the website you can decide whether to perform Full, Differential or Incremental backup.
```
Comment (hussainweb) [inline content/docs/common-areas/backups.md:25]:
```suggestion
In this type of backup all the information and resources of a website are copied over to the secondary target storage. The full backup ensures that files, databases, and scripts are backed up all the once. Generally, it is recommended to take the backup of the site at regular intervals. For example, some servers use automated scripts to perform the backup on a set day and time. Based on the website's scale, you can decide whether to perform a full, differential, or incremental backup.
```

**Review by hussainweb — CHANGES_REQUESTED** [review]

File: content/docs/common-areas/backups.md (hunk 6 above)
Comment (hussainweb) [reply content/docs/common-areas/backups.md:19]:
```suggestion
```

File: content/docs/common-areas/_index.md [hunk 8]
```diff
@@ -27,7 +27,9 @@ Most systems have multiple layers of caching, each of which would be suitable to
 
 APM or Application Performance Monitoring can be defined as a process or a practice of monitoring and tracking the performance of a software application using specific tools. The performance of a Drupal application can be monitored using various tools such as NewRelic, Elastic APM, Sentry etc..
 
-## Backups
+## [Backups]({{< relref "/docs/common-areas/backups" }}>)
```
Comment (hussainweb) [inline content/docs/common-areas/_index.md:30]:
```suggestion
## [Backups]({{< relref "/docs/common-areas/backups" >}})
```

**Pull request merged**